### PR TITLE
Ignore empty lines in the Procfile.

### PR DESCRIPTION
**Pull request by benmanns**
This pull request keeps the parser from raising an exception when it encounters a blank line.

### CODE REVIEW

File: lib/procfile_upstart_exporter/procfile_parser.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ def parse procfile
     ProcfileUpstartExporter.logger.debug 'Start parsing Procfile ' \
                                          "`#{ procfile }'"
     if File.exists? procfile
-      File.read(procfile).split("\n").map do |process_line|
+      File.read(procfile).split("\n").map(&:strip).reject(&:empty?).map do |process_line|
```
Comment (stupied4ever) [inline lib/procfile_upstart_exporter/procfile_parser.rb]: Can you please keep the 80 columns?

Comment (benmanns) [inline lib/procfile_upstart_exporter/procfile_parser.rb]: Sure. Would you prefer function composition on multiple lines? E.g.

```
File.read(procfile).split("\n").map(&:strip)
  .reject(&:empty?).map do |process_line|
...
```

Or assigning to intermediate variables? E.g.

```
lines = File.read(procfile).split("\n").map(&:strip)
lines.reject(&:empty?).map do |process_line|
...
```

Comment (stupied4ever) [inline lib/procfile_upstart_exporter/procfile_parser.rb]: That is up to you :)

I probably would make intermediate variables, but that is very personal

**Comment by stupied4ever** [comment]
Merged. Thx @benmanns

**Comment by leafac** [comment]
:heart: :yellow_heart: :green_heart: :purple_heart: :blue_heart:

**Pull request merged**